ts/image-occlusion/ImageOcclusionPage.svelte

```diff
@@ -6,18 +6,22 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
     import * as tr from "@tslib/ftl";
 
     import Container from "../components/Container.svelte";
+    import { addBrowserClasses } from "../reviewer/browser_selector";
     import { addOrUpdateNote } from "./add-or-update-note";
     import type { IOMode } from "./lib";
     import MasksEditor from "./MaskEditor.svelte";
     import Notes from "./Notes.svelte";
-    import StickyFooter from "./StickyFooter.svelte";
     import { hideAllGuessOne, textEditingState } from "./store";
 
     export let mode: IOMode;
 
     async function addNote(): Promise<void> {
         addOrUpdateNote(mode, $hideAllGuessOne);
     }
+    globalThis.imageOcclusion = {
```

Comment:
Please avoid polluting the global namespace. If you need to make something globally available, putting it in the top-level 'anki' would be better.